spec/lib/diaspora_federation/entities/comment_spec.rb

```diff
@@ -4,13 +4,15 @@ module DiasporaFederation
   describe Entities::Comment do
     let(:parent) { Fabricate(:post, author: bob) }
     let(:parent_entity) { Fabricate(:related_entity, author: bob.diaspora_id) }
-    let(:data) {
+    let(:parent_comment) { Fabricate(:comment)} 
+    let(:data) { 
       Fabricate
         .attributes_for(
           :comment_entity,
           author:      alice.diaspora_id,
           parent_guid: parent.guid,
-          parent:      parent_entity
+          parent:      parent_entity,
```

Comment:
Layout/HashAlignment: Align the keys and values of a hash literal if they span more than one line.